tests/integration/targets/lambda_layer/tasks/main.yml

```diff
@@ -45,7 +45,7 @@
         bucket: "{{ s3_bucket_name }}"
         mode: put
         object: "{{ s3_bucket_object }}"
-        permission: public-read
+        # permission: public-read  # Commented on because botocore.exceptions.ClientError: An error occurred (AccessControlListNotSupported) when calling the PutObject operation: The bucket does not allow ACLs
```

Comment:
Can you file an issue for us to come back and setup the bucket permissions correctly so that we can test the `permissions` key for objects?  I understand we want to unblock CI for the release in the short term but we should also fix this in the mid term.

Reply:
@jillr  Have filed this issue https://github.com/ansible-collections/amazon.aws/issues/1503